deploy/shakenfist_ci/tests/test_floating_ips.py

```diff
@@ -63,6 +63,17 @@ def test_simple(self):
         self.assertNotEqual(None, ifaces[0]['floating'])
 
         time.sleep(120)
-        r = requests.request('GET', 'http://%s/' % ifaces[0]['floating'])
-        self.assertEqual(200, r.status_code)
-        self.assertEqual('Floating IPs work!\n', r.text)
+
+        attempts = 0
+        for _ in range(6):
+            try:
+                r = requests.request(
+                    'GET', 'http://%s/' % ifaces[0]['floating'])
+                self.assertEqual(200, r.status_code)
+                self.assertEqual('Floating IPs work!\n', r.text)
+
+            except Exception as e:
+                if attempts < 5:
```

Comment:
probably should increment `attempts`

Reply:
Y U so hater? Also yes.

Reply:
Sorry to be a pain, but why it does it test 6 times regardless of the outcome?

Reply:
Because I am bad at computers.